app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml

```diff
@@ -8,24 +8,26 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <test name="GuestCheckoutWithMultipleProducts" extends="CreateConfigurableProduct">
+    <test name="RegisteredUserCheckoutWithMultipleProducts" extends="CreateConfigurableProduct">
```

Comment:
Did you intend to repurpose this test, or would there be value in running the same test for both guests and customers?

Reply:
Yes. For guest checkout there are 2 tests so this one  I repurposed for registered user.